src/styles/book.css

```diff
@@ -10,7 +10,41 @@
   grid-template-columns: 1.5fr 1fr 1fr;
   padding: 3vh 5vw;
   gap: 5px;
-  border-radius: 12px;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.progress-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.update-status-button {
+  background-color: blue;
+  border-radius: 3px;
+  color: #fff;
+  border: none;
+  padding: 1vh 1vw;
+  cursor: pointer;
+}
+
+.update-status-button:hover {
+  background-color: #fff;
+  box-shadow: 0 0 3px #000;
+  color: blue;
+}
+
+.update-status-button:active {
+  background-color: orange;
+  border: 1px solid #000;
+  box-shadow: 0 0 3px blue;
+  color: #fff;
+}
```

Comment:
- [ ] Kindly make sure you improve your design a little bit to stick as much as possible to the template design(e.g.: **used fonts(family, size, weight,...) on your text(book title & other text, ), width & border_color of the book item block, alignment of the book item block text,...**. Make sure you check all details from the **[template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)** to match the design.

### Template Design
![Screenshot from 2022-11-18 06-00-23](https://user-images.githubusercontent.com/83042742/202615949-c667b25c-8b9e-4e91-a63a-e736960d8f97.png)

### Your Design
![Screenshot from 2022-11-18 05-59-15](https://user-images.githubusercontent.com/83042742/202616066-375a2d1c-8ff5-405e-a15b-b5931883663e.png)

**But Good job with your design, it really looks perfect :+1: :+1: :100:**

Reply:
@tresorsawasawa Thank you for reviewing my project and appreciating the work done. I will improve on the styles asap